Index: src/main/resources/templates/fragment/footer.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/templates/fragment/footer.html b/src/main/resources/templates/fragment/footer.html
new file mode 100644
--- /dev/null	(date 1682648753284)
+++ b/src/main/resources/templates/fragment/footer.html	(date 1682648753284)
@@ -0,0 +1,7 @@
+<html>
+
+<footer th:fragment="footerFragment">
+  <hr>
+  copyright
+</footer>
+</html>
\ No newline at end of file
Index: src/main/resources/templates/layout/base.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/templates/layout/base.html b/src/main/resources/templates/layout/base.html
new file mode 100644
--- /dev/null	(date 1682656219998)
+++ b/src/main/resources/templates/layout/base.html	(date 1682656219998)
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="ko"
+      xmlns:layout="http://www.w3.org/1999/xhtml" xmlns:th="http://www.w3.org/1999/xhtml">
+<head>
+  <meta charset="UTF-8">
+  <title>Title</title>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
+  <link rel="stylesheet" type="text/css" th:href="@{/css/style.css}">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+</head>
+<body>
+  <!-- navi -->
+  <th:block th:replace="~{fragment/navi :: naviFragment}"></th:block>
+
+  <!-- content -->
+  <section>
+    <th:block layout:fragment="content"></th:block>
+  </section>
+
+  <!-- footer -->
+  <th:block th:replace="~{fragment/footer :: footerFragment}"></th:block>
+</body>
+</html>
\ No newline at end of file
Index: src/main/resources/templates/content/account/register.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/templates/content/account/register.html b/src/main/resources/templates/content/account/register.html
new file mode 100644
--- /dev/null	(date 1682656671244)
+++ b/src/main/resources/templates/content/account/register.html	(date 1682656671244)
@@ -0,0 +1,7 @@
+<html xmlns:layout="http://www.w3.org/1999/xhtml" layout:decorate="~{layout/base}">
+<div layout:fragment="content" id="index_container">
+  <h1>여기는</h1>
+  <hr>
+  <p>회원가입이여 ㅇㅋ?</p>
+</div>
+</html>
\ No newline at end of file
Index: src/main/resources/templates/content/index.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/templates/content/index.html b/src/main/resources/templates/content/index.html
new file mode 100644
--- /dev/null	(date 1682656133219)
+++ b/src/main/resources/templates/content/index.html	(date 1682656133219)
@@ -0,0 +1,7 @@
+<html xmlns:layout="http://www.w3.org/1999/xhtml" layout:decorate="~{layout/base}">
+<div layout:fragment="content" id="index_container">
+    <h1>여기다가</h1>
+    <hr>
+    <p>그거 잘 하면 됨 ㅇㅋ?</p>
+</div>
+</html>
\ No newline at end of file
Index: src/main/java/com/example/JakSim/Account/RegisterController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/JakSim/Account/RegisterController.java b/src/main/java/com/example/JakSim/Account/RegisterController.java
new file mode 100644
--- /dev/null	(date 1682656719137)
+++ b/src/main/java/com/example/JakSim/Account/RegisterController.java	(date 1682656719137)
@@ -0,0 +1,12 @@
+package com.example.JakSim.Account;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class RegisterController {
+    @GetMapping("/signup")
+    public String Signup(){
+        return "content/account/register";
+    }
+}
Index: src/main/java/com/example/JakSim/login/LoginController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/JakSim/login/LoginController.java b/src/main/java/com/example/JakSim/login/LoginController.java
new file mode 100644
--- /dev/null	(date 1682656744859)
+++ b/src/main/java/com/example/JakSim/login/LoginController.java	(date 1682656744859)
@@ -0,0 +1,11 @@
+package com.example.JakSim.login;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+@Controller
+public class LoginController {
+    @GetMapping("/login")
+    public String login(){
+        return "content/account/login";
+    }
+}
Index: src/main/java/com/example/JakSim/TstApplication.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.JakSim;\n\nimport org.springframework.boot.SpringApplication;\nimport org.springframework.boot.autoconfigure.SpringBootApplication;\n\n@SpringBootApplication\npublic class TstApplication {\n\n\tpublic static void main(String[] args) {\n\n\t\tSpringApplication.run(TstApplication.class, args);\n\t\t//Jorim\n\t\t// 왜이럼\n\t\t//test\n\t}\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/JakSim/TstApplication.java b/src/main/java/com/example/JakSim/TstApplication.java
--- a/src/main/java/com/example/JakSim/TstApplication.java	(revision 1b7985be68bc91e5386304cb2997d2c0ac051be5)
+++ b/src/main/java/com/example/JakSim/TstApplication.java	(date 1682490205855)
@@ -7,10 +7,6 @@
 public class TstApplication {
 
 	public static void main(String[] args) {
-
 		SpringApplication.run(TstApplication.class, args);
-		//Jorim
-		// 왜이럼
-		//test
 	}
 }
Index: src/main/resources/templates/fragment/navi.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/templates/fragment/navi.html b/src/main/resources/templates/fragment/navi.html
new file mode 100644
--- /dev/null	(date 1682657117076)
+++ b/src/main/resources/templates/fragment/navi.html	(date 1682657117076)
@@ -0,0 +1,33 @@
+<html xmlns:sec="http://www.w3.org/1999/xhtml" xmlns:th="http://www.w3.org/1999/xhtml">
+
+<nav class="navbar navbar-expand-lg bg-body-tertiary" th:fragment="naviFragment">
+  <div class="container-fluid">
+    <a class="navbar-brand" href="#">Navbar</a>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+        <li class="nav-item">
+          <a class="nav-link active" aria-current="page" href="/">Home</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#">Link</a>
+        </li>
+        <li class="nav-item dropdown">
+          <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+            Dropdown
+          </a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link disabled">Disabled</a>
+        </li>
+      </ul>
+      <form class="d-flex" role="search">
+        <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+        <button class="btn btn-outline-success" type="submit">Search</button>
+      </form>
+    </div>
+  </div>
+</nav>
+</html>
\ No newline at end of file
Index: src/main/java/com/example/JakSim/MainController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/JakSim/MainController.java b/src/main/java/com/example/JakSim/MainController.java
new file mode 100644
--- /dev/null	(date 1682655562487)
+++ b/src/main/java/com/example/JakSim/MainController.java	(date 1682655562487)
@@ -0,0 +1,13 @@
+package com.example.JakSim;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
+public class MainController {
+    @GetMapping("/")
+    public String index(){
+        return "content/index";
+    }
+}
Index: build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>plugins {\n\tid 'java'\n\tid 'org.springframework.boot' version '2.7.11'\n\tid 'io.spring.dependency-management' version '1.0.15.RELEASE'\n}\n\ngroup = 'com.example'\nversion = '0.0.1-SNAPSHOT'\nsourceCompatibility = '11'\n\nconfigurations {\n\tcompileOnly {\n\t\textendsFrom annotationProcessor\n\t}\n}\n\nrepositories {\n\tmavenCentral()\n}\n\ndependencies {\n\timplementation 'org.springframework.boot:spring-boot-starter-web'\n\tcompileOnly 'org.projectlombok:lombok'\n\truntimeOnly 'com.oracle.database.jdbc:ojdbc8'\n\tannotationProcessor 'org.projectlombok:lombok'\n\ttestImplementation 'org.springframework.boot:spring-boot-starter-test'\n}\n\ntasks.named('test') {\n\tuseJUnitPlatform()\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/build.gradle b/build.gradle
--- a/build.gradle	(revision 1b7985be68bc91e5386304cb2997d2c0ac051be5)
+++ b/build.gradle	(date 1682581627700)
@@ -2,11 +2,13 @@
 	id 'java'
 	id 'org.springframework.boot' version '2.7.11'
 	id 'io.spring.dependency-management' version '1.0.15.RELEASE'
+	id 'war'
 }
 
 group = 'com.example'
 version = '0.0.1-SNAPSHOT'
 sourceCompatibility = '11'
+apply plugin: 'war'
 
 configurations {
 	compileOnly {
@@ -23,6 +25,10 @@
 	compileOnly 'org.projectlombok:lombok'
 	runtimeOnly 'com.oracle.database.jdbc:ojdbc8'
 	annotationProcessor 'org.projectlombok:lombok'
+
+	implementation 'org.springframework.boot:spring-boot-starter-thymeleaf'
+	implementation 'nz.net.ultraq.thymeleaf:thymeleaf-layout-dialect'
+
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
 }
 
Index: src/main/resources/templates/content/account/login.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/templates/content/account/login.html b/src/main/resources/templates/content/account/login.html
new file mode 100644
--- /dev/null	(date 1682656419602)
+++ b/src/main/resources/templates/content/account/login.html	(date 1682656419602)
@@ -0,0 +1,7 @@
+<html xmlns:layout="http://www.w3.org/1999/xhtml" layout:decorate="~{layout/base}">
+<div layout:fragment="content" id="index_container">
+  <h1>여기가</h1>
+  <hr>
+  <p>로그인이여 ㅇㅋ?</p>
+</div>
+</html>
\ No newline at end of file
Index: src/main/resources/static/css/style.css
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/static/css/style.css b/src/main/resources/static/css/style.css
new file mode 100644
--- /dev/null	(date 1682656190558)
+++ b/src/main/resources/static/css/style.css	(date 1682656190558)
@@ -0,0 +1,3 @@
+section{
+    min-height: 500px;
+}
\ No newline at end of file
Index: src/main/resources/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/src/main/resources/application.properties b/src/main/resources/application.properties
--- a/src/main/resources/application.properties	(revision 1b7985be68bc91e5386304cb2997d2c0ac051be5)
+++ b/src/main/resources/application.properties	(date 1682559673902)
@@ -1,1 +1,1 @@
-
+spring.output.ansi.enabled=always
\ No newline at end of file
