Index: src/main/java/com/example/JakSim/MainController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.JakSim;\n\nimport org.springframework.stereotype.Controller;\nimport org.springframework.web.bind.annotation.GetMapping;\nimport org.springframework.web.bind.annotation.ResponseBody;\n\n@Controller\npublic class MainController {\n    @GetMapping(\"/\")\n    public String index(){\n        return \"content/index\";\n    }\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/JakSim/MainController.java b/src/main/java/com/example/JakSim/MainController.java
--- a/src/main/java/com/example/JakSim/MainController.java	(revision c6b3179c872fcaf16d22d8017db494ecfeadc8ad)
+++ b/src/main/java/com/example/JakSim/MainController.java	(date 1682669398061)
@@ -2,12 +2,12 @@
 
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.ResponseBody;
 
 @Controller
 public class MainController {
     @GetMapping("/")
-    public String index(){
+    public String index() {
+
         return "content/index";
     }
 }
Index: src/main/resources/static/css/style.css
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>section{\n    min-height: 500px;\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/static/css/style.css b/src/main/resources/static/css/style.css
--- a/src/main/resources/static/css/style.css	(revision c6b3179c872fcaf16d22d8017db494ecfeadc8ad)
+++ b/src/main/resources/static/css/style.css	(date 1682657556128)
@@ -1,3 +1,20 @@
-section{
-    min-height: 500px;
-}
\ No newline at end of file
+@font-face{
+    font-family: abster;
+    src: url(static/abster-webfont.woff) format('woff');
+}
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.container{
+    width: 100%;
+    height: 100vh; 
+    background: linear-gradient(rgba(0,0,0,0.8),rgba(0,0,0,0.8)),url(images/Group40267.png);
+}
+
+.container .header{
+    
+}
+
Index: src/main/resources/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>spring.output.ansi.enabled=always
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/application.properties b/src/main/resources/application.properties
--- a/src/main/resources/application.properties	(revision c6b3179c872fcaf16d22d8017db494ecfeadc8ad)
+++ b/src/main/resources/application.properties	(date 1682657811237)
@@ -1,1 +1,1 @@
-spring.output.ansi.enabled=always
\ No newline at end of file
+spring.output.ansi.enabled=always
Index: bin/main/templates/content/account/register.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/bin/main/templates/content/account/register.html b/bin/main/templates/content/account/register.html
new file mode 100644
--- /dev/null	(date 1682657545253)
+++ b/bin/main/templates/content/account/register.html	(date 1682657545253)
@@ -0,0 +1,7 @@
+<html xmlns:layout="http://www.w3.org/1999/xhtml" layout:decorate="~{layout/base}">
+<div layout:fragment="content" id="index_container">
+  <h1>여기는</h1>
+  <hr>
+  <p>회원가입이여 ㅇㅋ?</p>
+</div>
+</html>
\ No newline at end of file
Index: bin/main/templates/content/account/login.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/bin/main/templates/content/account/login.html b/bin/main/templates/content/account/login.html
new file mode 100644
--- /dev/null	(date 1682657545253)
+++ b/bin/main/templates/content/account/login.html	(date 1682657545253)
@@ -0,0 +1,7 @@
+<html xmlns:layout="http://www.w3.org/1999/xhtml" layout:decorate="~{layout/base}">
+<div layout:fragment="content" id="index_container">
+  <h1>여기가</h1>
+  <hr>
+  <p>로그인이여 ㅇㅋ?</p>
+</div>
+</html>
\ No newline at end of file
Index: bin/main/templates/content/index.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/bin/main/templates/content/index.html b/bin/main/templates/content/index.html
new file mode 100644
--- /dev/null	(date 1682657545254)
+++ b/bin/main/templates/content/index.html	(date 1682657545254)
@@ -0,0 +1,7 @@
+<html xmlns:layout="http://www.w3.org/1999/xhtml" layout:decorate="~{layout/base}">
+<div layout:fragment="content" id="index_container">
+    <h1>여기다가</h1>
+    <hr>
+    <p>그거 잘 하면 됨 ㅇㅋ?</p>
+</div>
+</html>
\ No newline at end of file
Index: .vscode/settings.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.vscode/settings.json b/.vscode/settings.json
new file mode 100644
--- /dev/null	(date 1682657556131)
+++ b/.vscode/settings.json	(date 1682657556131)
@@ -0,0 +1,3 @@
+{
+    "java.compile.nullAnalysis.mode": "automatic"
+}
\ No newline at end of file
Index: bin/main/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/bin/main/application.properties b/bin/main/application.properties
new file mode 100644
--- /dev/null	(date 1682657811237)
+++ b/bin/main/application.properties	(date 1682657811237)
@@ -0,0 +1,1 @@
+spring.output.ansi.enabled=always
Index: build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>plugins {\n\tid 'java'\n\tid 'org.springframework.boot' version '2.7.11'\n\tid 'io.spring.dependency-management' version '1.0.15.RELEASE'\n\tid 'war'\n}\n\ngroup = 'com.example'\nversion = '0.0.1-SNAPSHOT'\nsourceCompatibility = '11'\napply plugin: 'war'\n\nconfigurations {\n\tcompileOnly {\n\t\textendsFrom annotationProcessor\n\t}\n}\n\nrepositories {\n\tmavenCentral()\n}\n\ndependencies {\n\timplementation 'org.springframework.boot:spring-boot-starter-web'\n\tcompileOnly 'org.projectlombok:lombok'\n\truntimeOnly 'com.oracle.database.jdbc:ojdbc8'\n\tannotationProcessor 'org.projectlombok:lombok'\n\n\timplementation 'org.springframework.boot:spring-boot-starter-thymeleaf'\n\timplementation 'nz.net.ultraq.thymeleaf:thymeleaf-layout-dialect'\n\n\ttestImplementation 'org.springframework.boot:spring-boot-starter-test'\n}\n\ntasks.named('test') {\n\tuseJUnitPlatform()\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/build.gradle b/build.gradle
--- a/build.gradle	(revision c6b3179c872fcaf16d22d8017db494ecfeadc8ad)
+++ b/build.gradle	(date 1682657556122)
@@ -30,6 +30,10 @@
 	implementation 'nz.net.ultraq.thymeleaf:thymeleaf-layout-dialect'
 
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
+
+	implementation 'org.springframework.boot:spring-boot-starter-thymeleaf'
+	implementation 'nz.net.ultraq.thymeleaf:thymeleaf-layout-dialect'
+
 }
 
 tasks.named('test') {
Index: bin/main/templates/fragment/navi.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/bin/main/templates/fragment/navi.html b/bin/main/templates/fragment/navi.html
new file mode 100644
--- /dev/null	(date 1682657545254)
+++ b/bin/main/templates/fragment/navi.html	(date 1682657545254)
@@ -0,0 +1,33 @@
+<html xmlns:sec="http://www.w3.org/1999/xhtml" xmlns:th="http://www.w3.org/1999/xhtml">
+
+<nav class="navbar navbar-expand-lg bg-body-tertiary" th:fragment="naviFragment">
+  <div class="container-fluid">
+    <a class="navbar-brand" href="#">Navbar</a>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+        <li class="nav-item">
+          <a class="nav-link active" aria-current="page" href="/">Home</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#">Link</a>
+        </li>
+        <li class="nav-item dropdown">
+          <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+            Dropdown
+          </a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link disabled">Disabled</a>
+        </li>
+      </ul>
+      <form class="d-flex" role="search">
+        <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+        <button class="btn btn-outline-success" type="submit">Search</button>
+      </form>
+    </div>
+  </div>
+</nav>
+</html>
\ No newline at end of file
Index: bin/main/templates/fragment/footer.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/bin/main/templates/fragment/footer.html b/bin/main/templates/fragment/footer.html
new file mode 100644
--- /dev/null	(date 1682657545254)
+++ b/bin/main/templates/fragment/footer.html	(date 1682657545254)
@@ -0,0 +1,7 @@
+<html>
+
+<footer th:fragment="footerFragment">
+  <hr>
+  copyright
+</footer>
+</html>
\ No newline at end of file
Index: bin/main/static/css/style.css
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/bin/main/static/css/style.css b/bin/main/static/css/style.css
new file mode 100644
--- /dev/null	(date 1682657556128)
+++ b/bin/main/static/css/style.css	(date 1682657556128)
@@ -0,0 +1,20 @@
+@font-face{
+    font-family: abster;
+    src: url(static/abster-webfont.woff) format('woff');
+}
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.container{
+    width: 100%;
+    height: 100vh; 
+    background: linear-gradient(rgba(0,0,0,0.8),rgba(0,0,0,0.8)),url(images/Group40267.png);
+}
+
+.container .header{
+    
+}
+
Index: bin/main/templates/layout/base.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/bin/main/templates/layout/base.html b/bin/main/templates/layout/base.html
new file mode 100644
--- /dev/null	(date 1682657545254)
+++ b/bin/main/templates/layout/base.html	(date 1682657545254)
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="ko"
+      xmlns:layout="http://www.w3.org/1999/xhtml" xmlns:th="http://www.w3.org/1999/xhtml">
+<head>
+  <meta charset="UTF-8">
+  <title>Title</title>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
+  <link rel="stylesheet" type="text/css" th:href="@{/css/style.css}">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+</head>
+<body>
+  <!-- navi -->
+  <th:block th:replace="~{fragment/navi :: naviFragment}"></th:block>
+
+  <!-- content -->
+  <section>
+    <th:block layout:fragment="content"></th:block>
+  </section>
+
+  <!-- footer -->
+  <th:block th:replace="~{fragment/footer :: footerFragment}"></th:block>
+</body>
+</html>
\ No newline at end of file
